<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>743</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>931</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="search">
     <attribute name="title">
      <string>학생 조회 등록 삭제 수정</string>
     </attribute>
     <widget class="QFrame" name="searchFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>851</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="searchStudentIDText">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="searchYearComboBox">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="searchYear">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>학년</string>
       </property>
      </widget>
      <widget class="QComboBox" name="searchMajorComboBox">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>20</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>컴퓨터공학과</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>전자공학과</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>기계공학과</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="searchMajor">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>20</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>전공</string>
       </property>
      </widget>
      <widget class="QLabel" name="searchStudentID">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>학번</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="searchStudentNameText">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="searchStudentName">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="searchTable">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>721</width>
        <height>301</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <column>
       <property name="text">
        <string>학번</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>학년</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>전공</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>평균 학점</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>학생 리스트</string>
      </property>
     </widget>
     <widget class="QComboBox" name="searchSortComboBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>정렬 기준</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>학번</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>이름</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>학년</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>평균 학점</string>
       </property>
      </item>
     </widget>
     <widget class="QToolButton" name="searchAscPushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string> ↑</string>
      </property>
     </widget>
     <widget class="QToolButton" name="searchDescPushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>210</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchDeletePushButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>210</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>학생 삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStudentSearchButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>학생 조회</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchStudentAllSearchButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>전체 조회</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchRegistrationPushButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>30</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>학생 등록</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchUpdatePushButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>30</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>학생 수정</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>학생 조회 등록 삭제 수정</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addCourse">
     <attribute name="title">
      <string>과목 추가 삭제  및 학점 등록</string>
     </attribute>
     <widget class="QFrame" name="addCourseFrame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>851</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="courseStudentIDText">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="courseStudentID">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>학번</string>
       </property>
      </widget>
      <widget class="QLabel" name="courseCourseName">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>30</y>
         <width>61</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>수강 과목</string>
       </property>
      </widget>
      <widget class="QComboBox" name="courseCourseNameComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>30</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="courseMajor">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>30</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>전공</string>
       </property>
      </widget>
      <widget class="QComboBox" name="courseMajorComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>30</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>컴퓨터공학과</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>전자공학과</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>기계공학과</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="courseYear">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>학년</string>
       </property>
      </widget>
      <widget class="QComboBox" name="courseYearComboBox">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="courseName">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="courseNameText">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>81</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="courseSearchTable">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>801</width>
        <height>271</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>학번</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>학년</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>전공</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수강 과목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>학점</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>과목 추가 삭제  및 학점 등록</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>학생 리스트</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseSearchPushButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>학번 조회</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseAddPushButton">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>과목 추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseDeletePushButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>과목 삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseAllSearchPushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 조회</string>
      </property>
     </widget>
     <widget class="QPushButton" name="courseGradePushButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>학점 등록</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>학생 관리 프로그램</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
